CLAUDE.md

```
@@ -0,0 +1,51 @@
+  - Download and convert to MP3
+  - Track download progress
+  - Stream/download converted files
+
+- `packages/downloader/download.py`: Core functionality
+  - YouTube video download using yt-dlp
+  - Audio extraction with FFmpeg
+  - Metadata embedding with eyed3
+
+## Commands
+
+### Setup
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)